### PR TITLE
Google IAM verification tag

### PR DESCRIPTION
**Pull request by farinholt**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1"> 
+    <meta name="google-site-verification" content="NGcFSsc8cnsbnE_lbtEHYoWWSNe-o8S6S_ZhsFr5htI" />
```
Comment (stevecheckoway) [inline _includes/head.html]: `meta` elements are void elements, they don't allow self-closing start tags nor end tags.

Comment (farinholt) [reply _includes/head.html]: Oops :) I'll update that now

Comment (farinholt) [reply _includes/head.html]: Interestingly, that was the code Google provided me with...

Comment (stevecheckoway) [reply _includes/head.html]: It'd be necessary for XHTML. I forget what user agents are supposed to do with self-closed HTML elements (as opposed to foreign) but I suspect it's just to have a warning (that no browser would ever show to users).

**Pull request merged**